js/fetch_test.ts

```diff
@@ -47,24 +47,30 @@ testPerm({ net: true }, async function responseClone() {
   }
 });
 
-testPerm({ net: true }, async function fetchRequest() {
-  const addr = "127.0.0.1:4501";
+function bufferServer(addr: string): deno.Buffer {
   const listener = deno.listen("tcp", addr);
   const buf = new deno.Buffer();
   listener.accept().then(async conn => {
-    buf.readFrom(conn);
-    await conn.write(
+    const p1 = buf.readFrom(conn);
```

Comment:
I don't understand why this was necessary to "fix windows".
If this is needed to avoid an "invalid rid" error, that's a bug elsewhere, or...?

Reply:
Yes - it's a race condition that is present in the current build. It only seems to manifest itself on windows (and only sometimes)

Reply:
Then please add a comment explaining that.